### DISCUSSION TITLE
How to copy long fields of text?

### DISCUSSION THREAD

**Discussion by LetsPlayFetch** [Vector35/binaryninja-api#5480] [Q&A] [answered]
I'm new to Binary Ninja and having trouble copying large amounts of data. Previously, I was placing my cursor at the top of a function in pseudo code view, scrolling to the bottom, then clicking to highlight and copy the entire function. However, this method no longer works suddenly and I can only copy what is immediately visible on the screen. I'm hoping someone knows how I can 're enable' that feature or another solution for curser copying. I'm currently using the personal edition.

**Comment by psifertex** [comment, marked as answer]
I can't reproduce the behavior you're describing -- selecting large amounts of text off-screen still works for me. Is it related maybe to a specific file you're trying? What version of BN are you using? Does it happen on both dev and stable? There's no setting to change for that since we believe it should be working as you initally described. Nothing was intentionally changed about it.

That said, you can use `File` / `Export` / `Linear View` to dump the entire file to disk and snip out the pieces you want.

You might also find this example snippet useful: https://gist.github.com/psifertex/6fbc7532f536775194edd26290892ef7#file-export_to_text-py

**Reply by hxehex** [reply]
> I can't reproduce the behavior you're describing -- selecting large amounts of text off-screen still works for me.

I can reproduce the behavior. I can't copy an entire function, only what's visible on the screen. Latest Binja downloaded from the website, free, linux.
